native/src/routes/feedback/components/FeedbackModal.js

```diff
@@ -60,33 +84,34 @@ class FeedbackModal extends React.Component<PropsType> {
           <Description theme={theme}>{t('feedbackType')}</Description>
           <Picker
             selectedValue={feedbackOptions.indexOf(feedbackItem)}
-            onValueChange={this.props.onFeedbackOptionChanged}
+            onValueChange={props.onFeedbackOptionChanged}
             mode='dropdown'>
             {feedbackOptions.map((item, index) => (
               <Picker.Item label={item.label} value={index} key={index} />
             ))}
           </Picker>
+
           <Description theme={theme}>
             {' '}
             {isPositiveFeedback ? t('positiveComment') : t('negativeComment')}
             {!isPositiveFeedback && <RequiredText>*</RequiredText>}
           </Description>
-          <Input
-            theme={theme}
-            onChangeText={this.props.onCommentChanged}
-            autoFocus
-            value={comment}
-            multiline
-            placeholderTextColor={theme.colors.textSecondaryColor}
-            placeholder={t('yourFeedback')}
-          />
+          <Input theme={theme} onChangeText={props.onCommentChanged} value={comment} multiline />
+
+          <DescriptionContainer theme={theme}>
+            <Description theme={theme}>{t('contactMailAddress')}</Description>
+            <OptionalText>({t('optionalInfo')})</OptionalText>
+          </DescriptionContainer>
+
+          <Input theme={theme} onChangeText={props.onFeedbackContactMailChanged} value={contactMail} />
```

Comment:
can you add some distance between this text and the button? I think it looks really weird on android

Reply:
done